### DISCUSSION TITLE
Group of TextBox auto adjust the Width when edited

### DISCUSSION THREAD

**Discussion by chaimel** [fabricjs/fabric.js#9643] [Q&A] [answered]
In a group I edit the textbox and tried to setCoords() to update the new width of the group

var clickedObject = event.target;
      if (clickedObject) {
        if (clickedObject.isType('group')) {
           // Check if the entire group is locked
      var isGroupLocked = clickedObject.locked;

      console.log('Is the group locked?', isGroupLocked);
          const target = event?.subTargets[0]
          if(target){
            window.svgCanvas.setActiveObject(target)
            window.creativeLab.showFloatToolBox();
            if (event.e.detail === 2) {
              if(target.type === "textbox"){
                target.enterEditing()
             
              }
              
              target.setCoords();
              clickedObject.setCoords();
              window.svgCanvas.renderAll();
              window.svgCanvas.requestRenderAll();

              // Listen for the text editing to be exited
              target.on('editing:exited', function () {
                clickedObject.setCoords();
                window.svgCanvas.renderAll();
                window.svgCanvas.requestRenderAll();
              });
            } 
          }
          
        } else {
          console.log('Clicked on a single object:', clickedObject)
        }
      }

![Capture](https://github.com/fabricjs/fabric.js/assets/57937227/08f3446f-86db-4e2d-a4a2-c8efd63d4d94)

**Comment by chaimel** [comment]
Im using v5

**Comment by ShaMan123** [comment, marked as answer]
call `Textbox#initDimensions`
if you wish the group to resize to fit the content I advice upgrading to v6
